integration-tests/utils/integration-helpers.js

```diff
@@ -31,7 +32,7 @@ const makeWorkspace = withSignedInPage(async ({ page, billingProject }) => {
   try {
     const response = await page.evaluate(async (name, billingProject) => {
       try {
-        return await window.Ajax().Workspaces.create({ namespace: billingProject, name, attributes: {} })
+        return await window.Ajax().Workspaces.create({ namespace: billingProject, name, attributes: {}, bucketLocation: 'us-central1' })
```

Comment:
My understanding is that rawls is being updated so that workspaces will be created with us-central1 buckets by default. Is that correct? So I'm guessing that this code is added because that rawls change isn't live yet. I'd suggest adding a comment here explaining why we need to specify the bucketLocation at this point.

Reply:
This is a good point-- can we just rely on the Rawls default instead of duplicating it in both repos?

Reply:
True, I'll wait until that is merged and remove this change.

Reply:
FWIW, we are putting this back in to be consistent with the fact that for non-test usage we do pass the bucketLocation instead of relying on the Rawls default.

But @aherbst-broad, can you reference the default location constant instead of explicitly putting `us-central1`?

Reply:
Following up: we are blocked on shipping the rawls PR due to a downstream consumer needing to make a change. Until that ships I'm going to explicitly set the region on the IT helper. I've added a comment to that effect here as well.

Reply:
To be consistent with the fact that we explicitly pass it for non-test usage, I think it is fine to always pass the default value. But I think you should use the constant.

Reply:
As discussed below, we can't reference the desired constant from this module, I've restored the hardcoded string here.